resources/views/includes/tabs.blade.php

```diff
@@ -1,58 +1,51 @@
+<div x-data="{ tab: @entangle('activeTab') }" class="lfb-tabs-wrapper">
+    <div class="lfb-tab-container">
         {{-- Tabs Links --}}
-        <aside class="py-4 lg:col-span-2">
-            <div class="flex flex-shrink-0 items-center px-6 mb-6">
-                <div class="text-3xl font-semibold text-c_gray-800">{{ $formTitle }}</div>
+        <aside class="lfb-tab-nav-wrapper">
+            <div class="lfb-tab-nav-container">
+                <div class="lfb-tab-nav-title-wrapper">
+                    <div class="lfb-tab-nav-title">{{ $formTitle }}</div>
+                </div>
+                <nav class="lfb-tab-nav-links" x-model="tab">
+                    @foreach($fields as $field)
+                        <div class="lfb-tab-nav-link-item">
+                            <a x-bind:class="[ tab == '{{ $field['tab']['key'] }}' ? 'lfb-tab-nav-link-active' : '']" class="lfb-tab-nav-link" x-on:click.prevent="tab='{{ $field['tab']['key'] }}'">
+                                {{ $field['tab']['title'] }}
+                            </a>
+                        </div>
+                    @endforeach
+                </nav>
             </div>
-            <nav x-model="tab">
-                @foreach($fields as $field)
-                    <li x-bind:class="[ tab == '{{ $field['tab']['key'] }}' ? 'bg-c_gray-100 border-c_gray-300' : '']" class="text-c_gray-800 border-transparent hover:bg-c_gray-200/50 hover:border-c_gray-300 border-l-4 px-3 py-2 flex items-center text-sm font-medium cursor-pointer">
-                        <a x-on:click.prevent="tab='{{ $field['tab']['key'] }}'">
-                            {{ $field['tab']['title'] }}
-                        </a>
-                    </li>
-                @endforeach
-            </nav>
         </aside>
 
         {{-- Tabs Content --}}
-        <div class="lg:col-span-10 relative" wire:target="save" wire:loading.class="opacity-50">
-            <div class="relative">
+        <div class="lfb-tab-content-wrapper" wire:target="save" wire:loading.class="lfb-tab-content-wrapper-loading-class">
+            <div class="lfb-tab-content-container">
                 @error('tabWarning')
-                    <div class="shadow mx-4 mt-6 overflow-hidden rounded-md" role="alert">
-                        <div class="flex">
-                            <div class="bg-cyan-500 w-14 text-center p-2">
-                                <div class="flex justify-center h-full items-center">
-                                    <x-heroicon-o-exclamation-circle  class="h-6 w-6 text-white" />
-                                </div>
+                    <div class="lfb-tab-error-wrapper" role="alert">
+                        <div class="lfb-tab-error-container">
+                            <div class="lfb-tab-error-icon flex justify-center items-center">
+                                <x-heroicon-o-exclamation-circle  />
```

Comment:
could you please replace it with svg with ability to customize error message icon?

Reply:
The feature has been implemented.